db/products.js

```diff
@@ -1,21 +1,75 @@
-const {client} = require('./client')
+const {client} = require('./client');
 
 const getAllProducts = async () => {
-    try {
-      const { rows: products } = await client.query (`
-      SELECT *
-      FROM products;
-      `);
+  try {
+    const { rows: products } = await client.query (`
+    SELECT *
+    FROM products;
+    `);
+    return products;
+  }catch (error){
+    throw error;
+  }
+};
 
-      return products;
-  
-    } catch (error) {
-  
-      throw error;
-    }
-  
-}
+const getOrderProductById = async ({id}) => {
```

Comment:
will you move all of the functions you wrote into a new file like order_products? since products and order_products are not one in the same, we should have these broken apart for consistency throughout the project.

Reply:
also I think this should be (id) rather than ({id}) based on the instructions but I could be wrong